cdap-ui/app/cdap/components/Administration/AdminConfigTabContent/SystemProfilesAccordion/index.js

```diff
@@ -111,3 +108,23 @@ export default class SystemProfilesAccordion extends Component {
     );
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    // We have to get profiles count from ownProps to show on the accordion label, instead
+    // of getting from store state, if ProfilesStore is not initialized. This store is only
+    // initalized when ProfilesListView component is mounted, which only happens when the
+    // accordion is expanded
+    profilesCount: state.profiles.length || ownProps.profiles.length
```

Comment:
Hhmm this might cause uncertainty. If we need the profiles count why not trigger an action to populate the `ProfilesStore` and use it as the source of truth? Does it have to the `ProfilesListView` that populates the store?